fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/data/SavingsProductDataValidator.java

```diff
@@ -340,6 +343,22 @@ public void validateForCreate(final String json) {
             baseDataValidator.reset().parameter(minBalanceForInterestCalculationParamName).value(minBalanceForInterestCalculation)
                     .ignoreIfNull().zeroOrPositiveAmount();
         }
+        Boolean isLienAllowed = this.fromApiJsonHelper.parameterExists(lienAllowedParamName, element);
```

Comment:
I don't think this is correct because you're only checking whether the parameter exists instead of the parameter's value. Am I missing something?